src/main/java/minegame159/meteorclient/modules/movement/Flight.java

```diff
@@ -46,16 +46,18 @@
             .build()
     );
 
+    // Anti Kick
+
     private final Setting<AntiKickMode> antiKickMode = sgGeneral.add(new EnumSetting.Builder<AntiKickMode>()
-            .name("anti-kick-mode")
+            .name("mode")
             .description("The mode for anti kick.")
             .defaultValue(AntiKickMode.Packet)
             .build()
     );
 
     private final Setting<Integer> delay = sgAntiKick.add(new IntSetting.Builder()
             .name("delay")
-            .description("The amount of time, in ticks, between toggles in normal mode.")
+            .description("The amount of time in ticks between toggles in normal mode.")
```

Comment:
This fix is wrong.

Reply:
How so? Commas are unneccessary here

Reply:
Not true, that is a subordinate clause and so requires commas. It adds extra information but is not required to make the sentence make sense. Because it is in the middle, it is an embedded clause and so needs commas on each side.

Reply:
they aren't really

Reply:
Try saying that sentence with the pauses. It sounds kinda dumb out loud and I've never seen anybody use commas in that way to talk or write

Reply:
I'm telling you how the English language works. Idc how you think it sounds, those are the rules.

Reply:
You seem to really like making stuff grammatically correct and if you want to be grammatically correct then these commas and the other commas you removed are required.